src/Form/test-cases/ign_signup.html

```diff
@@ -0,0 +1,24 @@
+<!-- https://www.ign.com/ -->
+<form action="#"><label id="nickname-label" for="nickname" class="jsx-4229970651"><span
+            class="label-text">Username</span><span class="supplemental jsx-2108932070">Already a user? <button
+                class="jsx-1500221710">Sign In</button></span><input name="nickname" id="nickname" type="text"
+            class="jsx-1151081807" value="" data-manual-scoring="username"></label><label id="email-label" for="email"
```

Comment:
Should this be `username` or `unknown`? The signup form has a username and an email input; the login form accepts either one.

Reply:
I'd go for `unknown` in this case, I think. If the login form accepts both it's somewhat irrelevant, but `unknown` sounds more accurate. Thanks.